### PR TITLE
Copy explanation for centralised mentoring notes from Slack

### PR DESCRIPTION
**Pull request by SaschaMann**
Not really sure where this belongs 🤷

### CODE REVIEW

File: contributing/website-copy.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copy for the Exercism website
+
+Repo: <https://github.com/exercism/website-copy/>
+
+This document is obviously incomplete, it merely serves as a way to save a question & answer from the Slack paywall.
```
Comment (SaschaMann) [inline contributing/website-copy.md]:
```suggestion
This document is obviously incomplete, right now it merely serves as a way to save a question & answer from the Slack paywall.
```

**Comment by iHiD** [comment]
I think this probably needs to end up in the `product` section, or a specific section where we explain **why** things are how they are. I'll leave this PR open until we've got a final arrangement for this repo then we can move it to the correct place. Thanks for opening it.

**Comment by SaschaMann** [comment]
> I think this probably needs to end up in the product section, or a specific section where we explain why things are how they are

Perhaps it could be useful to have a FAQ-like section that we could link to from [chestertons-fence.md](https://github.com/exercism/docs/blob/main/contributing/chestertons-fence.md) that covers a bunch of these whys.

**Comment by ErikSchierboom** [comment]
I think adding a document to the https://github.com/exercism/docs/tree/main/building/product section could work well, as the whole goal of the documents in there is to explain why things are as they are.

**Comment by SaschaMann** [comment]
@ErikSchierboom This PR was mainly there for archival purposes. Feel free to turn it into a proper article or tag it as help-wanted for someone to take over :)

**Comment by ErikSchierboom** [comment]
Okay, will do!

**Pull request closed without merging**